docs/github-action.md

```diff
@@ -47,9 +47,39 @@ jobs:
 
 As written, the scanner will run on 12:12 pm UTC every Monday. You can change the schedule by following the instructions [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule).
 
+### Customization
+
+`osv-scanner-reusable.yml` takes the `scan-args` parameter as input, this value is passed to `osv-scanner` CLI after being split by each line. See the [usage]()./usage) page for the available options.
+
+Importantly `--format` and `--output` flags are already set by the reusable workflow and should not be overridden here.
```

Comment:
I think it would also be good practice to explain the defaults.

Reply:
```suggestion
Importantly `--format` and `--output` flags are already set by the reusable workflow and should not be overridden here.

The default behavior will recursively scan your repository from the root, but skip commit scanning to focus on dependencies.
```

Reply:
Added defaults value as a code block, and added inline comments describing what each line does